realm/realm-library/src/main/java/io/realm/DynamicRealm.java

```diff
@@ -279,6 +279,22 @@ static DynamicRealm createInstance(RealmConfiguration configuration) {
     }
 
     /**
+     * Returns a distinct set of objects from a specific class that fulfill the multiple fields
+     * condition. If the result is sorted, the first object of a column will be returned in case of
+     * multiple occurrences, otherwise it is undefined which object is returned.
```

Comment:
I would also add a description for the behavior for multiple fields. Perhaps something like:

"If multiple distinct fields are given, all _distinct_ combinations of these fields will be returned."